core/math/math.odin

```diff
@@ -34,8 +37,8 @@ MAX_F64_PRECISION :: 16 // Maximum number of meaningful digits after the decimal
 MAX_F32_PRECISION ::  8 // Maximum number of meaningful digits after the decimal point for 'f32'
 MAX_F16_PRECISION ::  4 // Maximum number of meaningful digits after the decimal point for 'f16'
 
-RAD_PER_DEG :: TAU/360.0
-DEG_PER_RAD :: 360.0/TAU
+RAD_PER_DEG :: TAU/360.0 // Being used to convert degrees to radians. (see `math.to_radians`)
```

Comment:
For documentation, although sounds counterintuitive, you should duplicate it rather than making references to other points of documentation. For constants of the kind `X_PER_Y` I suggest you show how to use these constants to perform the conversion.